sse_client/event.go

```diff
@@ -0,0 +1,119 @@
+package sse_client
```

Comment:
Becouse we have multiple instances of marathon-consul but we want only one of them to register services. When leader abdicate it connection will be closed and we will once again connect and check if we are connected to master.

Any idea how to do it better?